resources/user-agents/browsers/safari/safari-5-2.json

```diff
@@ -27,43 +27,26 @@
       },
       "children": [
         {
-          "match": "Mozilla/5.0 (SMART-TV; X11;#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Version/#MAJORVER#.#MINORVER#* Safari/*",
-          "device": "Smart-TV",
-          "platforms": [
-            "Linux_64", "Linux_32", "Linux"
-          ]
-        },
-        {
-          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Version/#MAJORVER#.#MINORVER#* Safari/*",
-          "device": "Linux Desktop",
-          "platforms": [
-            "CentOS", "Linux_64", "Linux_32", "Linux"
-          ]
-        },
-        {
-          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Version/#MAJORVER#.#MINORVER#* Safari/*",
-          "device": "Windows Desktop",
-          "platforms": [
-            "Win2000", "WinXPa_64", "WinXPa_32", "WinXPb_64", "WinXPb_32", "Vista_64", "Vista_32",
-            "Win7_64", "Win7_32", "Win8_64", "Win8_32",
-            "Win8_1_64", "Win8_1_32"
-          ]
-        },
-        {
           "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Version/#MAJORVER#.#MINORVER#* Safari/*",
           "device": "Macintosh",
           "platforms": [
-            "OSX_10_4", "OSX_10_5", "OSX_10_6", "OSX_10_7", "OSX_10_8", "OSX_10_9", "OSX_10_10",
+            "OSX_10_9", "OSX_10_8", "OSX_10_7", "OSX_10_6",
             "OSX"
-          ]
+          ],
+          "properties": {
```

Comment:
It also wasn't released for Mac according to wikipedia, but I did find this article that discusses it being released as a Beta: https://www.macrumors.com/2012/02/16/apple-seeds-safari-5-2-for-os-x-lion-to-developers/ So I left it in and just marked it as a beta.